### PR TITLE
Add v6.0 & dev-master to wordpress-stubs version constraints

### PR DESCRIPTION
**Pull request by joemaller**
Added constraints to allow this to be installed alongside the recently updated wordpress-stubs package. Also added dev-master since these are for testing and some people prefer to always have the latest release.

### CODE REVIEW

**Comment by szepeviktor** [comment]
Thank you for your PR!
I'm sorry to report that I'm an old, rule-based man 👴🏻 

Please remove `dev-master` then I'll accept it.

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "homepage": "https://github.com/php-stubs/acf-pro-stubs",
     "license": "GPL-2.0-or-later",
     "require": {
-        "php-stubs/wordpress-stubs": "^4.7 || ^5.0"
+        "php-stubs/wordpress-stubs": "^4.7 || ^5.0 || ^6 || dev-master"
```
Comment (szepeviktor) [inline composer.json]:
```suggestion
        "php-stubs/wordpress-stubs": "^4.7 || ^5.0 || ^6.0"
```

**Comment by szepeviktor** [comment]
Thank you @joemaller!

If you need the cutting edge you may still publish a very simple package depending on `"php-stubs/wordpress-stubs": "dev-master"` - but it hurts more than it helps.

**Comment by szepeviktor** [comment]
@joemaller You can find your change in new releases :)
https://github.com/php-stubs/acf-pro-stubs/tags

**Comment by joemaller** [comment]
Excellent, thank you. Apologies for trying to "sneak" in the `dev-master` thing. ☺️

**Pull request merged**